tests/e2e/advanced_shared_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This test suite includes test cases to verify advanced shared volume
+// cases including multiple writers writing data to the same volume
+
+// +build runonceshared
+
+package e2e
+
+import (
+	"github.com/vmware/vsphere-storage-for-docker/tests/utils/dockercli"
+	"github.com/vmware/vsphere-storage-for-docker/tests/utils/inputparams"
+	"github.com/vmware/vsphere-storage-for-docker/tests/utils/misc"
+	"github.com/vmware/vsphere-storage-for-docker/tests/utils/verification"
+	. "gopkg.in/check.v1"
+)
+
+const (
+        // Data that will be written to the test file in shared volume
+        data = "1234567890ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+        // Name of the test file in shared volume
+        testFileName = "test.txt"
+)
+
+type AdvancedSharedTestSuite struct {
+	config          *inputparams.TestConfig
+	esx             string
+	mgr1            string
```

Comment:
I think what I mean in other swarm related test for vDVS, the common configuration that we have is 1 master + 2 workers. Can you test run with that configuration? If it is yes, let us use the common configuration for swarm test.